app/processing/payment-request/schema.js

```diff
@@ -4,7 +4,9 @@ const { DAX_CODES } = require('../../constants/schedules')
 
 module.exports = Joi.object({
   paymentRequestId: Joi.number().integer().required(),
+  agreementNumber: Joi.string().required(),
```

Comment:
Missed in previous PR, but `dueDate` is a string in the format `DD/MM/YYYY`

Reply:
Bumpin' if has been missed

Reply:
Still dnt see this as resolved, see here for ref: https://github.com/DEFRA/ffc-pay-statement-constructor/blob/PAY-2318-aggregate-map-and-build-statement-constructor-object/test/mock-components/mock-due-date.js